ext/zend_test/test.stub.php

```diff
@@ -156,6 +156,13 @@ public function call(): void {}
         public static function callStatic(): void {}
     }
 
+    /** @data-class */
+    class ZendTestBox {
```

Comment:
Can't this be `data class ZendTestBox`, without then also requiring the introduction of the new `@data-class` annotation?

Reply:
Ultimately yes, but https://github.com/nikic/PHP-Parser/ doesn't support this syntax yet, for obvious reasons. ^^